### PR TITLE
feat: sorting on pools

### PR DESCRIPTION
**Pull request by SarjuHansaliya**
- added sorting & Pool Weights
sorting

- show APR in mobile
apr_mobile

### CODE REVIEW

File: src/pages/Earn/index.tsx [hunk 1]
```diff
@@ -47,31 +60,88 @@ export default function Earn({
...
+        })
+        .sort(function(info_a, info_b) {
+          if (sortBy.field === 'totalStakedInWavax') {
+            if (sortBy.desc) {
+              return info_a.totalStakedInWavax?.greaterThan(info_b.totalStakedInWavax ?? JSBI.BigInt(0)) ? -1 : 1
+            } else {
+              return info_a.totalStakedInWavax?.lessThan(info_b.totalStakedInWavax ?? JSBI.BigInt(0)) ? -1 : 1
+            }
+          }
+          if (sortBy.field === 'multiplier') {
```
Comment (bmino) [inline src/pages/Earn/index.tsx:103]: would these values be better as constant variables (enums)?

File: src/state/stake/hooks.ts [hunk 2]
```diff
@@ -59,548 +59,654 @@ export interface Staking {
...
-    version: 1
+    version: 1,
+    multiplier: 0
   },
   WAVAX_SHIBX_V1: {
     tokens: [WAVAX[ChainId.AVALANCHE], SHIBX[ChainId.AVALANCHE]],
     stakingRewardAddress: '0x0029381eFF48E9eA963F8095eA204098ac8e44B5',
-    version: 1
+    version: 1,
+    multiplier: 0
```
Comment (bmino) [inline src/state/stake/hooks.ts:431]: shouldn't be 0

**Review by bmino — CHANGES_REQUESTED** [review]
those hardcoded multipliers should be updated. some of the newer pools have 0's

File: src/pages/Earn/index.tsx (hunk 1 above)
Comment (SarjuHansaliya) [reply src/pages/Earn/index.tsx]: fixed.

File: src/state/stake/hooks.ts (hunk 2 above)
Comment (SarjuHansaliya) [reply src/state/stake/hooks.ts]: fixed.

**Review by bmino — APPROVED** [review]

**Pull request merged**